ignite/pkg/proxy/proxy.go

```diff
@@ -1,4 +1,4 @@
-package cosmoscmd
+package proxy
```

Comment:
Any particular for changing package?

`proxy` has the same problem. We would want project to use it (to launch chain under GItpod) but without importing entire CLI codebase

Reply:
This file import many other `ignite/pkg` packages, so I found simpler for the moment to move it out of `cosmoscmd`, into its own package. Therefore, `cosmoscmd` doesn't depend any more on ignite CLI.

But this is a temporary fix.

> I would consider it as part of https://github.com/ignite/cli/issues/2658 but maybe not fix the issue, as I think one of the issue was not only the circular dep but also the fact that projects may want more customization with the entire command app code in their codebase

got it!  I remove the Fix.

Reply:
This is obsolete